### ISSUE TITLE
[Question] ASNetworkImageNode is downloading images all the time..

### ISSUE THREAD

**Issue by MrS3** [facebook/AsyncDisplayKit#741]
Hi
How to prevent ASNetworkImageNode from downloading image all the time ? 
I have ASTableView ,  custom node cell and ASNetworkImageNode();
During the scrolling nodes are downloading images again and again witch cause memory increasing.
I thought ASNetworkImageNode downloading image only once.
I don't know how to cache , store this images ? Should I use ASImageCacheProtocol some how ?
Can you guys help me with this ? Please forgive me if this is not a right place to ask this kind of question. I understand that its not an issue.
Regards
Piotr

**Comment by appleguy** [comment]
@MrS3 naturally a cache is necessary if you want images to be stored for much longer than the time they are onscreen.  Caching has costs — either memory, or on disk space, and have different performance characteristics — so it wouldn't be reasonable for ASDK to try to solve that problem in a generic way.  Apps have /extremely/ different needs when it comes to memory usage and caching.

You have a good lead on the answer...the ASImageCacheProtocol is provided as a surprisingly easy way to plug into the ASDK components and implement any caching strategy you would like.  I believe somewhere in the framework there is a default / basic image cache you can use, but this may not have been finished.

Thanks for posting and please do open another issue if you have a more specific question on integrating with the caching protocol!  Also welcome any PRs implementing a simple NSCache-based image cache if the framework doesn't currently provide one.
